test/jsx.js

```diff
@@ -0,0 +1,79 @@
+/**
```

Comment:
@balthazzar /test directory is intended only for unit test files, otherwise it would be executed by mocha. If this file is still needed move it to another directory.